src/custom/state/price/middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Middleware, isAnyOf } from '@reduxjs/toolkit'
+import { AppState } from 'state'
+import { initialPriceLoadAnalytics } from 'utils/analytics'
+import * as PriceActions from './actions'
+
+const isUpdateQuoteAction = isAnyOf(PriceActions.updateQuote)
+
+export const priceMiddleware: Middleware<Record<string, unknown>, AppState> = (store) => (next) => (action) => {
+  const state = store.getState()
+
+  if (isUpdateQuoteAction(action)) {
+    if (!state.price.initialQuoteLoaded) {
```

Comment:
it works, i would probably have created a mutable flag. This way you don't need even to have `initialQuoteLoaded` in the state. You created the flag just for this analytics event, right? 

nit: can you add a clarifying comment, like:
// Send an event only on the first price load

Reply:
Added a comment